src/main/java/mate/academy/spring/controller/AuthenticationController.java

```diff
@@ -0,0 +1,29 @@
+package mate.academy.spring.controller;
+
+import mate.academy.spring.mapper.impl.response.UserResponseMapper;
+import mate.academy.spring.model.User;
+import mate.academy.spring.model.dto.request.UserRequestDto;
+import mate.academy.spring.model.dto.response.UserResponseDto;
+import mate.academy.spring.security.AuthenticationService;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class AuthenticationController {
+    private final AuthenticationService authenticationService;
+    private final UserResponseMapper responseMapper;
```

Comment:
We should follow DI principle and use an abstraction here
```suggestion
    private final DtoResponseMapper<UserResponceDto, User> responseMapper;
```
![image](https://github.com/mate-academy/jv-spring-rest-web-practice/assets/96497090/9fb87ab5-3c6e-41fa-bc55-59153d469033)

Please, fix it in all controllers